src/System.Diagnostics.DiagnosticSource/src/System.Diagnostics.DiagnosticSource.builds

```diff
@@ -2,8 +2,10 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="System.Diagnostics.DiagnosticSource.csproj" />
+    <Project Include="System.Diagnostics.DiagnosticSource.csproj"/>
```

Comment:
Agreed I wouldn't add a TargetGroup for this first entry which is an empty/default TargetGroup.